src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -893,6 +893,86 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
     return {{element::f32, element::f32}};
 }
 
+/// TANH ///
+jit_tanh_emitter::jit_tanh_emitter(dnnl::impl::cpu::aarch64::jit_generator *host,
+                                   dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                   const std::shared_ptr<ov::Node>& node)
+                                   : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+    sigmoid_emitter = std::make_unique<jit_sigmoid_emitter>(h, host_isa, node);
+}
+
+jit_tanh_emitter::jit_tanh_emitter(dnnl::impl::cpu::aarch64::jit_generator *host,
+                                   dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                   const ov::element::Type exec_prc)
+                                   : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+    sigmoid_emitter = std::make_unique<jit_sigmoid_emitter>(h, host_isa, exec_prc);
+}
+
+size_t jit_tanh_emitter::get_inputs_count() const { return 1; }
+
+size_t jit_tanh_emitter::get_aux_vecs_count() const {
+    return sigmoid_emitter->get_aux_vecs_count() + 1;
+}
+
+size_t jit_tanh_emitter::get_aux_gprs_count() const {
+    return sigmoid_emitter->get_aux_gprs_count() + 1;
+}
+
+void jit_tanh_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_tanh_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (exec_prc_ != ov::element::f32) {
+        OPENVINO_THROW("unsupported precision: " + exec_prc_.to_string());
+    }
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    TReg src = TReg(in_vec_idxs[0]);
+    TReg dst = TReg(out_vec_idxs[0]);
+
+    TReg aux = TReg(aux_vec_idxs[sigmoid_emitter->get_aux_vecs_count()]);
+
+    h->ld1r(aux.s, table_val2("two"));
+    h->uni_fmul(aux.s, src.s, aux.s);
+
+    std::vector<size_t> sigmoid_aux_vec_idxs;
+    sigmoid_aux_vec_idxs.assign(aux_vec_idxs.begin(), aux_vec_idxs.end());
+    std::vector<size_t> sigmoid_aux_gpr_idxs = { aux_gpr_idxs[0] };
```

Comment:
I might miss something, but sigmoid_aux_vec_idxs and sigmoid_aux_gpr_idxs seem not used.

Reply:
fixed, thanks

Reply:
cleared, thanks